amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -138,18 +141,25 @@ export class TableDetail extends React.Component<
       title: `Columns (${this.props.tableData.columns.length})`,
     });
 
-    // Dashboard content
-    tabInfo.push({
-      content: (
-        <ResourceList
-          allItems={this.props.tableData.dashboards}
-          itemsPerPage={DASHBOARDS_PER_PAGE}
-          source={TABLE_SOURCE}
-        />
-      ),
-      key: 'dashboards',
-      title: `Dashboards (${this.props.tableData.dashboards.length})`,
-    });
+    if (indexDashboardsEnabled()) {
+      const loadingTitle = (
```

Comment:
This logic supports:
1. Not showing this tab at all if dashboards are not enabled.
2. Indicating the loading state then showing the total:
![dashboard_tab_loading](https://user-images.githubusercontent.com/1790900/85486956-829d2700-b580-11ea-87ee-31ff65083207.gif)